manpages/lumina-screenshot.8

```diff
@@ -0,0 +1,18 @@
+.Dd October 20, 2017
+.Dt LUMINA-SCREENSHOT 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-screenshot
+.Nd ("PrintScreen" button by default) creates, modifies, and saves screenshots.
+
+.Sh SYNOPSIS
+.Nm
+
+.Sh DESCRIPTION
+.Nm
+is a tool that allows the user to take a snapshot of their screen. It then allows the user to view, crop, zoom, and save that screenshot. The user can also choose to open the screenshot with a different application. There is also an option to copy, allowing the user to easily copy and paste the screenshot. Under the settings tab the screenshot tool has options to capture the entire session, a single screen, a single window, or a selected area.
```

Comment:
"that allows the user" is unnecessary -> .Nm takes screenshots of connected monitors. The user can view, crop zoom, and save...